tasks/compile.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const fs = require('fs');
+const { promisify } = require('util');
+const path = require('path');
+const uglifyJs = require('uglify-js');
+
+const readFile = promisify(fs.readFile);
+const packageJson = require('../package');
+
+const baseTemplates = {
+  'oc-template-handlebars': {
+    externals: [
+      {
+        global: 'Handlebars',
+        url: 'https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.7.7/handlebars.runtime.min.js'
+      }
+    ]
+  },
+  'oc-template-jade': {
+    externals: [
+      {
+        global: 'jade',
+        url: 'https://cdnjs.cloudflare.com/ajax/libs/jade/1.11.0/runtime.min.js'
+      }
+    ]
+  },
+  'oc-template-es6': { externals: [] }
+};
+
+module.exports = async function compile(conf = {}) {
+  const registeredTemplates = { ...baseTemplates, ...conf.templates };
+
+  const version = packageJson.version;
+  const srcPath = '../src/';
+  const vendorPath = '../vendor/';
+  const licenseLink =
+    'https://github.com/opencomponents/oc-client-browser/tree/master/LICENSES';
+  const license = `/*! OpenComponents client v${version} | (c) 2015-${new Date().getFullYear()} OpenTable, Inc. | ${licenseLink} */`;
```

Comment:
```suggestion
  const license = `/*! OpenComponents client v${version} | (c) 2015-${new Date().getFullYear()} OpenComponents community | ${licenseLink} */`;
```

Reply:
Done. To be honest I didn't merge this yet because if it's going to be useful. My idea was to have a way to compile on the fly the oc-client so either the registry can save a new version with more templates, or really serve on the fly per request, but this breaks two things.

- oc-client-browser now it's updated only when "oc" package gets updated.
- when you ask for registryUrl/oc-client/client.js, it redirects you to your storage path, which potentially could be a CDN, and it's cached for a year.

If we make it so that the registry can override this, then either
- On startup it will have to always override the client.js, and therefore invalidate the 1year cache header.
- Per request it will have to, instead of redirecting to the storage/cdn, serve it on the fly, which means that you would lose CDN benefits.

So still not sure what's a good way to solve having extra templates on your client.js by default being served on the default endpoint of `registryUrl/oc-client/client.js`.